content/en/integrations/guide/prometheus-metrics.md

```diff
@@ -7,65 +7,138 @@ further_reading:
 - link: "https://www.datadoghq.com/blog/sql-server-metrics/#create-a-stored-procedure-to-generate-and-collect-metrics"
   tag: "Blog"
   text: "Create a stored procedure to generate and collect metrics"
+- link: "/integrations/openmetrics/"
+  tag: "Documentation"
+  text: "Learn about the OpenMetrics integration"
 - link: "/integrations/mysql/"
   tag: "Documentation"
-  text: "Datadog-MySQL integration"
+  text: "Learn about the MySQL integration"
 - link: "/agent/kubernetes/prometheus/"
   tag: "Documentation"
   text: "Kubernetes Prometheus and OpenMetrics metrics collection"
 ---
 
-If you are using Datadog's Prometheus or OpenMetrics checks, you may be interested in how these metrics map to existing Datadog metrics types.
+## Overview
 
-See [Kubernetes Prometheus and OpenMetrics Metrics Collection][1] for more information.
+This page walks you through how Prometheus or OpenMetrics check metrics map to existing Datadog metric types.
 
 ## Prometheus and OpenMetrics metric types
 
-* `counter`: a cumulative metric that represents a single monotonically increasing counter, whose value can only increase—or be reset to zero.
-* `gauge`: a metric that represents a single numeric value, which can arbitrarily go up and down.
-* `histogram`: samples observations and counts them in configurable buckets; also provides a sum of all observed values.
-* `summary`: similar to histogram; samples observations, provides a sum of all observed values, and calculates configurable quantiles over a sliding time window.
+For more information, see [Kubernetes Prometheus and OpenMetrics Metrics Collection][1].
```

Comment:
I'm not sure this link fits with the topic, at least not as the first sentence in a section about the types, since the target of the link doesn't seem to even mention metric types at all.

Reply:
```suggestion
```